modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -94,6 +94,25 @@ enum class GShape: int
     GFRAME,
 };
 
+namespace gapi {
+namespace s11n {
+struct IOStream;
+struct IIStream;
+
+namespace detail {
+template<typename T, typename U> struct wrap_serialize {
+    static std::function<void(gapi::s11n::IOStream&, const util::any&)> serialize;
```

Comment:
What's the reason for this field to be static?

Reply:
Probably you've got the idea of this function wrong..

Reply:
Removed